umbraco/Umbraco-CMS
# V14: refactor property editor aliases

# Notes
- Refactored richtext editors `TinyMCE` alias to `RichText` instead, to not leak technology

# How to test
- Be on v14/dev
- Create a document type with a rich text property (allow as root)
- Go to the template and insert the RichText `@Model.Value("{YourPropertyAliasHere}")`
- Create some content with that document type (type in some text in the rich text editor)
- Assert that you can see the frontend (localhost/44331) and the things you have typed in the rich text editor
- Swap to this branch
- It should still work 👍
